tests/frontend-storage/cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
This is very similar to utils/common-config/cypress.config.ts. I think the common config could be imported from common utils package and only the `specPattern` could be overwritten